htdocs/core/modules/modWebhook.class.php

```diff
@@ -0,0 +1,542 @@
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * 	\defgroup   webhook     Module Webhook
+ *  \brief      Webhook module descriptor.
+ *
+ *  \file       htdocs/webhook/core/modules/modWebhook.class.php
+ *  \ingroup    webhook
+ *  \brief      Description and activation file for module Webhook
+ */
+include_once DOL_DOCUMENT_ROOT.'/core/modules/DolibarrModules.class.php';
+
+/**
+ *  Description and activation class for module Webhook
+ */
+class modWebhook extends DolibarrModules
+{
+	/**
+	 * Constructor. Define names, constants, directories, boxes, permissions
+	 *
+	 * @param DoliDB $db Database handler
+	 */
+	public function __construct($db)
+	{
+		global $langs, $conf;
+		$this->db = $db;
+
+		// Id for module (must be unique).
+		// Use here a free id (See in Home -> System information -> Dolibarr for list of used modules id).
+		$this->numero = 68305; // TODO Go on page https://wiki.dolibarr.org/index.php/List_of_modules_id to reserve an id number for your module
+
+		// Key text used to identify module (for permissions, menus, etc...)
+		$this->rights_class = 'webhook';
+
+		// Family can be 'base' (core modules),'crm','financial','hr','projects','products','ecm','technic' (transverse modules),'interface' (link with external tools),'other','...'
+		// It is used to group modules by family in module setup page
+		$this->family = "interface";
+
+		// Module position in the family on 2 digits ('01', '10', '20', ...)
+		$this->module_position = '90';
+
+		// Gives the possibility for the module, to provide his own family info and position of this family (Overwrite $this->family and $this->module_position. Avoid this)
+		//$this->familyinfo = array('myownfamily' => array('position' => '01', 'label' => $langs->trans("MyOwnFamily")));
+		// Module label (no space allowed), used if translation string 'ModuleWebhookName' not found (Webhook is name of module).
+		$this->name = preg_replace('/^mod/i', '', get_class($this));
+
+		// Module description, used if translation string 'ModuleWebhookDesc' not found (Webhook is name of module).
+		$this->description = "WebhookDescription";
+		// Used only if file README.md and README-LL.md not found.
+		$this->descriptionlong = "WebhookDescription";
+
+		// Author
+		$this->editor_name = 'Editor name';
+		$this->editor_url = 'https://www.example.com';
+
+		// Possible values for version are: 'development', 'experimental', 'dolibarr', 'dolibarr_deprecated' or a version string like 'x.y.z'
+		$this->version = 'dolibarr';
```

Comment:
Because this is a first version of module, we must first start with version = "development"